tests/rptest/tests/mirror_maker_test.py

```diff
@@ -160,12 +162,16 @@ def test_consumer_group_mirroring(self, source_type):
                                          log_level="TRACE")
         self.mirror_maker.start()
 
-        msg_cnt = 100
+        # 10k messages of size 2**15 is ~312MB of data.
+        # 100 messages of size 2**9 is ~50KB of data.
+        msg_size = 2**15 if self.redpanda.dedicated_nodes else 2**9
+        msg_cnt = 10000 if self.redpanda.dedicated_nodes else 100
```

Comment:
10k messages of 2**15 size is the largest amount of data confirmed to work on CDT so far.

10k messages of 2**19 size (i.e., ~4GB of data) was attempted on CDT but we ran into space issues such as
`OpenJDK 64-Bit Server VM warning: Insufficient space for shared memory file:`
I'll try that size again on a fresh cluster in the morning.